dattri/algorithm/influence_function.py

```diff
@@ -201,59 +209,416 @@ def lissa_collate_fn(
         )
 
 
-class IFAttributorDataInf(BaseInnerProductAttributor):
-    """The inner product attributor with DataInf inverse hessian transformation."""
+class IFAttributorDataInf(BaseAttributor):
```

Comment:
Maybe it's better to still inherit from BaseInnerProductAttributor and just overwrite some of the member functions, given a lot of member functions are the same here.

Reply:
Fixed.